code/game/machinery/computer/camera.dm

```diff
@@ -245,6 +245,16 @@
 	light_color = "#3848B3"
 	light_power_on = 0.5
 
+obj/machinery/computer/security/wooden_tv/public
```

Comment:
```suggestion
/obj/machinery/computer/security/wooden_tv/public
```

Reply:
Derp.  I thought I'd fixed that.  Thank you!